src/initial_conditions/atmos_state.jl

```diff
@@ -69,6 +70,14 @@ energy_variables(ls, ::TotalEnergy) = (;
     )
 )
 
+atmos_surface_field(surface_space, config, ::NoSurfaceTemperature) = 
+    (;)
+function atmos_surface_field(surface_space, config, ::PrognosticSurfaceTemperature)
+    surface_coords = Fields.coordinate_field(surface_space)
+    #default prognostic surface temperature set to 300, could change later 
+    return (; sfc = map(_ -> (; T = eltype(config)(300)), surface_coords))
```

Comment:
Is `config` here only for getting FT? @dennisYatunin Could we get FT from `surface_space`, so we don't need to pass in `config`?